returnn/datasets/meta.py

```diff
@@ -197,6 +198,8 @@ def __init__(self,
     :param str|None seq_order_control_dataset: if set, this dataset will define the order for each epoch.
     :param str|None seq_lens_file: filename. json. dict[str,dict[str,int]], seq-tag -> data-key -> len.
       Use if getting sequence length from loading data is too costly.
+    :param str|None default_data_key: if seq_order_control_dataset is not set, this will determine which dataset
```

Comment:
Do you really allow `None`?

Reply:
No, this is a leftover from an earlier approach, this will be removed.